tests/scripts/arm32_ci_test.sh

```diff
@@ -57,6 +60,12 @@ do
             exit_with_error "--buildConfig can be Debug, Checked or Release" true
         fi
         ;;
+    --ryujit)
+        if [[ "$__buildConfig" != "Release" ]]; then
+            exit_with_error "--ryujit is only avalable for Release by now" true
+        fi
```

Comment:
Is this test required? It seems like this script should accept `--ryujit` unconditionally, and it's the job of the caller to only pass it for tests that will succeed.

Reply:
I see. I just want to prevent other CI accidentally enabled for ryujit CI, because only release build is stable for Ryujit ARM32 when writing this script.

I will accept `--ryujit` uncondionally.